src/app/components/layout-sidebar/layout-sidebar-demo.component.ts

```diff
@@ -5,5 +5,19 @@ import { Component, OnInit } from '@angular/core';
   templateUrl: './layout-sidebar-demo.component.html'
 })
 export class StacheLayoutSidebarDemoComponent implements OnInit {
+  public routes: any[] = [
```

Comment:
Does this have to be any?  Or do we have an interface of routes we can type this?